packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -6023,23 +6023,23 @@ describe('ReactDOMFizzServer', () => {
     expect(reportedServerErrors.length).toBe(1);
     expect(reportedServerErrors[0].message).toBe('Oops!');
 
+    const reportedCaughtErrors = [];
     const reportedClientErrors = [];
     ReactDOMClient.hydrateRoot(container, <App />, {
+      onCaughtError(error) {
```

Comment:
I'm kind of confused why the lack of this override didn't fail the test before because it should've logged to console.error which fails the test.